### PR TITLE
Metronome configured to use Telegraf

### PR DESCRIPTION
**Pull request by kensipe**
## High-level description

What features does this change enable? What bugs does this change fix?

## Corresponding DC/OS tickets (obligatory)

These DC/OS JIRA ticket(s) must be updated (ideally closed) in the moment this PR lands:

  - [DCOS_OSS-4935](https://jira.mesosphere.com/browse/DCOS_OSS-4935) Configure metrics exporter for Metronome.

## Checklist for all PRs

  - [x] Added a comprehensible changelog entry to `CHANGES.md` or explain why this is not a user-facing change:
  - [ ] Included a test which will fail if code is reverted but test is not. If there is no test please explain here:
  - [x] Read the [DC/OS contributing guidelines](https://github.com/dcos/dcos/blob/master/contributing.md)
  - [x] Followed relevant code rules [Rules for Packages and Systemd](https://github.com/dcos/dcos/tree/master/docs)

## Checklist for component/package updates:

If you are changing components or packages in DC/OS (e.g. you are bumping the sha or ref of anything underneath `packages`), then in addition to the above please also include:

  - [ ] Change log from the last version integrated (this should be a link to commits for easy verification and review): 
  - [ ] Test Results: [link to CI job test results for component]
  - [ ] Code Coverage (if available): [link to code coverage report]

### CODE REVIEW

**Comment by kensipe** [comment]
@mesosphere-mergebot label ready for review

File: gen/dcos-config.yaml [hunk 1]
```diff
@@ -1961,6 +1961,9 @@ package:
       LIBPROCESS_PORT=15201
       METRONOME_SCHEDULER_TASK_LAUNCH_CONFIRM_TIMEOUT=600000
       METRONOME_SCHEDULER_TASK_LAUNCH_TIMEOUT=600000
+      METRONOME_STATSD_HOST="localhost"
+      METRONOME_STATSD_PORT=61825
```
Comment (timcharper) [inline gen/dcos-config.yaml]: This is the static port assigned for STATSD? Isn't 32000+ reserved for dynamic ports?

Comment (kensipe) [reply gen/dcos-config.yaml]: jira:  says this is it:  https://jira.mesosphere.com/browse/DCOS_OSS-4935
same for marathon https://jira.mesosphere.com/browse/DCOS-47693
and what was used for the Marathon configuration:  https://github.com/dcos/dcos/pull/4323/files#diff-3f4e5f3f735bf0976a529aae710020bbR15

Double checking...

Comment (kensipe) [reply gen/dcos-config.yaml]: yep...  https://github.com/dcos/dcos/blob/master/docs/admin-ports.md
```
UDP
53: dcos-net (dns)
61825: dcos-telegraf```

Comment (timcharper) [reply gen/dcos-config.yaml]: Ah, Linux uses 32768-61000. So, this is not in the dynamic port range. I learned (or probably relearned) something. Everything looks good!

**Pull request merged**